src/MSBuild/XMake.cs

```diff
@@ -2071,6 +2077,11 @@ bool recursing
                         interactive = ProcessBooleanSwitch(commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.Interactive], defaultValue: true, resourceName: "InvalidInteractiveValue");
                     }
 
+                    if (commandLineSwitches[CommandLineSwitches.ParameterlessSwitch.GraphBuild])
+                    {
+                        enableGraphBuild = true;
```

Comment:
Make this a parameterized switch and call `ProcessBooleanSwitch()` so that users can have the feature enabled/disabled in a response file but override the value at the command-line